src/Orchard.Environment.Extensions.Abstractions/IExtensionPriorityStrategy.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Orchard.Environment.Extensions.Features;
+
+namespace Orchard.Environment.Extensions
+{
+    public interface IExtensionPriorityStrategy
+    {
+        double GetPriority(IFeatureInfo feature);
```

Comment:
`int`, just because we use it for all other priorities in the system, so does MVC. Use numbers that are separate enough that one can add a new strategy between some other existing ones.